spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RewriteDataFilesProcedure.java

```diff
@@ -140,36 +140,55 @@ private RewriteDataFiles checkAndApplyOptions(InternalRow args, RewriteDataFiles
     return action.options(options);
   }
 
-  private RewriteDataFiles checkAndApplyStrategy(RewriteDataFiles action, String strategy, SortOrder sortOrder) {
+  private RewriteDataFiles checkAndApplyStrategy(
+      RewriteDataFiles action,
+      String strategy,
+      String sortOrderString,
+      Schema schema) {
+    List<ExtendedParser.RawOrderField> zOrderFields = Lists.newArrayList();
```

Comment:
Could just be a list of Zorder terms at the moment since you check the type before adding to this list and the other fields are meaningless of RawOrderField in this case